website/content/api-docs/secret/ldap.mdx

```diff
@@ -63,6 +63,9 @@ to search and change entry passwords in LDAP.
 - `client_tls_cert` `(string: <optional>)` - Client certificate to provide to the LDAP server, must be x509
   PEM encoded.
 - `client_tls_key` `(string: <optional>)` - Client key to provide to the LDAP server, must be x509 PEM encoded.
+- `skip_static_role_import_rotation` `(string: <boolean>)` - The default value to use for `skip_import_rotation` when
```

Comment:
It looks like boolean type fields are usually described like `(boolean: false)`. It also communicates the default if not provided, so we can probably remove the "This value defaults to false." as well?

Reply:
wow, i'm not sure where (string: <boolean>) came from.

I changed it to (bool: <optional>) just because that's the format used in another parameter above (starttls and insecure_tls)